app/controllers/assignments_controller.rb

```diff
@@ -64,6 +64,11 @@ def destroy
     end
   end
 
+  def desktop
+    code = current_user.api_token
```

Comment:
Using `code` here to refer to the temporary token that's passed before the app sends the actual `access_token` over. Following the terminology that we use for auth on [dotcom](https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps/#2-users-are-redirected-back-to-your-site-by-github).

Not sure if it could be confusing, what do you think?